probe-rs-debug/src/exception_handling/riscv.rs

```diff
@@ -0,0 +1,100 @@
+use std::ops::ControlFlow;
+
+use crate::{
+    DebugError, DebugInfo, DebugRegisters, StackFrame,
+    exception_handling::{ExceptionInfo, ExceptionInterface},
+    unwind_pc_without_debuginfo,
+};
+
+use probe_rs::{MemoryInterface, RegisterRole, RegisterValue};
+
+pub struct RiscvExceptionHandler;
+
+impl ExceptionInterface for RiscvExceptionHandler {
+    fn exception_details(
```

Comment:
Aren't those the defaults on the trait or could be made such?

Reply:
Sure why not